js/service/shell/input-box.js.rej

```diff
@@ -0,0 +1,15 @@
+***************
+*** 198,200 ****
+  });
+  
+  exports.onInput = onInput;
+--- 198,206 ----
+  });
+  
+  exports.onInput = onInput;
++ exports.done = function() {
++   tty.print('\n');
++   inputEnabled = true;
++   drawPrompt();
++   drawCursor();
++ };
```

Comment:
@kesla I accidentally deleted some pushed work, so I got the patch from github and patched it to put it back. I forgot to delete it.